Cargo.toml

```diff
@@ -20,6 +20,7 @@ harness = false
 
 [dependencies]
 async-lock = "2.6"
+cfg-if = "1"
```

Comment:
(polling already depends on cfg-if, so this doesn't increase number of dependencies)